data/config.json

```diff
@@ -201,9 +201,10 @@
             },
             "filters": {
               "sampleRate": 100,
-              "maxSize": 3,
+              "maxSize": 1000,
```

Comment:
would the test fail with 3?

Reply:
I left that in accidentally when testing. I'll remove.